src/In.ProjectEKA.HipService/Gateway/GatewayPathConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace In.ProjectEKA.HipService.Gateway
+{
+    public class GatewayPathConstants
+    {
+        public static string OnDiscoverPath = "/care-contexts/on-discover";
+        public static string OnLinkInitPath = "/v1/links/link/on-init";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5570468)